Cargo.toml

```diff
@@ -20,10 +20,10 @@ path = "src/main.rs"
 doc = false
 
 [dependencies]
-rand = "^0.3.14"
+rand = "0.5.5"
 http-muncher = "^0.3"
-regex = "^0.2.2"
-lazy_static = "^0.2"
+regex = "1.0.5"
```

Comment:
please preserve the `^` (here and everywhere else), so we can still be flexible about the allowed version. see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html for more details.